lua/cobalt2/theme.lua

```diff
@@ -66,3 +66,11 @@ Group.new("healthError", colors.red:light(), nil, nil)
 Group.new("healthSuccess", colors.green, nil, nil)
 Group.new("healthWarning", colors.light_yellow, nil, nil)
 Group.new("qfLineNr", colors.light_grey, colors.cobalt_bg, nil)
+-- Dashboard
+Group.new("DashboardShortCut", colors.cyan, colors.cobalt_bg, nil)
+Group.new("DashboardHeader", colors.blue, colors.cobalt_bg, nil)
+Group.new("DashboardCenter", colors.magenta, colors.cobalt_bg, nil)
+Group.new("DashboardFooter", colors.cyan, colors.cobalt_bg, nil)
+Group.new("DashboardKey", colors.yellow, colors.cobalt_bg, nil)
+Group.new("DashboardDesc", colors.light_green, colors.cobalt_bg, nil)
+Group.new("DashboardIcon", colors.yellow, colors.cobalt_bg, styles.bold)
```

Comment:
Please keep these in a separate file as it is being done for so many other plugins. You can create a file in `plugins/` folder name `dashboard-nvim.lua` and put these there.